docs/api/breaking-changes.md

```diff
@@ -8,6 +8,27 @@ The `FIXME` string is used in code comments to denote things that should be fixe
 
 # Planned Breaking API Changes (5.0)
 
+## `Services` MenuItem creation
+
+```js
+// Deprecated
+const menuOld = Menu.buildFromTemplate([
+  {
+    role: 'services',
+    submenu: []
+  }
+])
+Menu.setApplicationMenu(menuOld)
+
+// Replace with
+const menuNew = Menu.buildFromTemplate([
+  {
+    role: 'services'
```

Comment:
Why this is a breaking change? Will the old syntax blow up an app in version 5?

Reply:
☝️ I don't think the old syntax will stop working right?